diff -ur mauikit-imagetools-3.1.0/src/CMakeLists.txt mauikit-imagetools-3.1.0/src/CMakeLists.txt
--- mauikit-imagetools-3.1.0/src/CMakeLists.txt	2024-02-27 18:28:41.400097391 +0100
+++ mauikit-imagetools-3.1.0/src/CMakeLists.txt	2024-02-27 18:29:55.009874825 +0100
@@ -26,6 +26,7 @@
     code/exiv2extractor.h
     code/geolocation/city.h
     code/geolocation/cities.h
+    code/geolocation/kdtree.hpp
     code/moduleinfo.h
     code/textscanner.h)
 
