--- okular-26.04.3/CMakeLists.txt	2026-08-05 12:14:39.217651401 +0200
+++ okular-26.04.3/CMakeLists.txt	2026-08-05 12:15:56.519136063 +0200
@@ -32,6 +32,7 @@
 include(CMakePackageConfigHelpers)
 include(ECMSetupQtPluginMacroNames)
 include(ECMAddAndroidApk)
+include(ECMQmlModule)
 
 if (ANDROID AND (FORCE_NOT_REQUIRED_DEPENDENCIES STREQUAL ""))
     set(FORCE_NOT_REQUIRED_DEPENDENCIES "KF6Wallet;KF6DocTools;Qt6Qml;TIFF;LibSpectre;KExiv2Qt6;DjVuLibre;EPub;Discount;")
