From 90c62ef78693d4c8f54329a791e053f61a01ddc8 Mon Sep 17 00:00:00 2001
From: Camilo Higuita <chiguitar@unal.edu.co>
Date: Mon, 9 Sep 2024 10:00:51 -0500
Subject: [PATCH] remove reference to setFetchRelations

---
 src/code/incidencewrapper.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/code/incidencewrapper.cpp b/src/code/incidencewrapper.cpp
index f5232f7..55c70f2 100644
--- a/src/code/incidencewrapper.cpp
+++ b/src/code/incidencewrapper.cpp
@@ -36,7 +36,6 @@ IncidenceWrapper::IncidenceWrapper(QObject *parent)
     Akonadi::ItemFetchScope scope;
     scope.fetchFullPayload();
     scope.fetchAllAttributes();
-    scope.setFetchRelations(true);
     scope.setAncestorRetrieval(Akonadi::ItemFetchScope::Parent);
     setFetchScope(scope);
     
-- 
GitLab

