From f1ad116a2333b2052c69a326f0d5872b84fbd219 Mon Sep 17 00:00:00 2001
From: Luigi311 <git@luigi311.com>
Date: Wed, 1 Feb 2023 17:18:46 -0700
Subject: [PATCH] Increase burst minimum (MR 28)

---
 src/io_pipeline.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/io_pipeline.c b/src/io_pipeline.c
index 8434420..b720e03 100644
--- a/src/io_pipeline.c
+++ b/src/io_pipeline.c
@@ -385,12 +385,12 @@ capture(MPPipeline *pipeline, const void *data)
                 info->camera, V4L2_CID_EXPOSURE_AUTO, V4L2_EXPOSURE_MANUAL);
 
         // Get current gain to calculate a burst length;
-        // with low gain there's 2, with the max automatic gain of the ov5640
+        // with low gain there's 3, with the max automatic gain of the ov5640
         // the value seems to be 248 which creates a 5 frame burst
         // for manual gain you can go up to 11 frames
         gain = mp_camera_control_get_int32(info->camera, V4L2_CID_GAIN);
         gain_norm = (float)gain / (float)info->gain_max;
-        burst_length = (int)fmax(sqrt(gain_norm) * 10, 1) + 1;
+        burst_length = (int)fmax(sqrt(gain_norm) * 10, 2) + 1;
         captures_remaining = burst_length;
 
         // Change camera mode for capturing
-- 
GitLab

